WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -727,6 +727,229 @@ - (UIView *)tableView:(UITableView *)tableView viewForFooterInSection:(NSInteger
+                                                       callback:^{
+        [weakSelf showThemes];
+    }];
+    row.quickStartIdentifier = QuickStartTourElementThemes;
+    return row;
+}
+
+- (BlogDetailsRow *)menuRow
+{
+    __weak __typeof(self) weakSelf = self;
+    BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Menus", @"Menus option in the blog details")
+                                                          image:[[UIImage gridiconOfType:GridiconTypeMenus] imageFlippedForRightToLeftLayoutDirection]
+                                                       callback:^{
+        [weakSelf showMenus];
+    }];
+    return row;
+}
+
+- (BlogDetailsRow *)domainsRow
+{
+    __weak __typeof(self) weakSelf = self;
+    BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Domains", @"Noun. Title. Links to the Domains screen.")
+                                                     identifier:BlogDetailsSettingsCellIdentifier
+                                        accessibilityIdentifier:@"Domains Row"
+                                                          image:[UIImage gridiconOfType:GridiconTypeDomains]
+                                                       callback:^{
+        [weakSelf showDomainsFromSource:BlogDetailsNavigationSourceRow];
+    }];
+    return row;
+}
+
+- (BlogDetailsRow *)siteSettingsRow
+{
+    __weak __typeof(self) weakSelf = self;
+    BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Site Settings", @"Noun. Title. Links to the blog's Settings screen.")
+                                                     identifier:BlogDetailsSettingsCellIdentifier
+                                        accessibilityIdentifier:@"Settings Row"
+                                                          image:[UIImage gridiconOfType:GridiconTypeCog]
+                                                       callback:^{
+        [weakSelf showSettingsFromSource:BlogDetailsNavigationSourceRow];
+    }];
+    return row;
+}
+
+- (BlogDetailsRow *)adminRow
+{
+    __weak __typeof(self) weakSelf = self;
+    BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:[self adminRowTitle]
+                                                          image:[UIImage gridiconOfType:GridiconTypeMySites]
+                                                       callback:^{
+        [weakSelf showViewAdmin];
+        [weakSelf.tableView deselectSelectedRowWithAnimation:YES];
+    }];
+    UIImage *image = [[UIImage gridiconOfType:GridiconTypeExternal withSize:CGSizeMake(BlogDetailGridiconAccessorySize, BlogDetailGridiconAccessorySize)] imageFlippedForRightToLeftLayoutDirection];
+    UIImageView *accessoryView = [[UIImageView alloc] initWithImage:image];
+    accessoryView.tintColor = [WPStyleGuide cellGridiconAccessoryColor]; // Match disclosure icon color.
+    row.accessoryView = accessoryView;
+    row.showsSelectionState = NO;
+    return row;
+}
```

Comment:
These "row" properties are extracted from the existing `sectionViewModel` methods. The definition of these rows was embedded in those methods and I needed to re-use the same rows in the new `sectionViewModel [methods I added.](https://github.com/wordpress-mobile/WordPress-iOS/pull/20476/files#r1157860414)